.travis.yml

```diff
@@ -34,11 +34,10 @@ script:
         for FILE in scripts/*; do
             rm $BINDIR/$(basename $FILE)
         done
-        # Run the "make check" per each commit origin..HEAD
+        # Run the "make check" per each commit in PR (TRAVIS_COMMIT_RANGE)
         ERR=""
-        MASTER=$(git rev-parse origin/master)
-        echo Master is $MASTER
-        for COMMIT in $(git rev-list origin..HEAD | tail -n+2); do
+        echo Branch is $TRAVIS_BRANCH
```

Comment:
This is only a debug info, but it could be useful at least in the beginning... Let's keep it here.